boot/freeldr/freeldr/include/arch/pc/x86common.h

```diff
@@ -13,8 +13,8 @@
 #define STACK16ADDR         HEX(6F00) /* The 16-bit stack top will be at 0000:6F00 */
 #define BSS_START           HEX(6F00)
 #define STACKLOW            HEX(7000)
-#define STACKADDR           HEX(F000) /* The 32/64-bit stack top will be at 0000:F000, or 0xF000 */
-#define FREELDR_BASE        HEX(F800)
+#define STACKADDR           HEX(E000) /* The 32/64-bit stack top will be at 0000:E000, or 0xE000 */
```

Comment:
So this also reduces the size of the stack available for Freeldr to run.

Reply:
Yeah. but i'm testing yet.

Reply:
I reverted because making the frldr16 with 4kb would break multiboot.

Reply:
base address change will come in another PR